examples/interop-examples/README.md

```diff
@@ -0,0 +1,102 @@
+# interop examples
+
+**WIP**
+
+## Run
+```
+cd examples/interop-examples
+npm i
+npm start
+```
+
+Open http://127.0.0.1:8080 in your favorite browser.
+
+## Tutorial
+
+Steps
+1. create IPFS instance
+
+First, we need an IPFS instance that we'll use throughout the tutorial. For the sake of convinience, we'll use `ipfs-daemon` module to create and IPFS instance.
```

Comment:
We should not be recommending `ipfs-daemon` in order to instantiate `js-ipfs`, it just means that `js-ipfs` needs to improve its init process.